src/node/desktop/scripts/copy-files.ts

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
You don't need this script. See other comment about using files in-place for dev builds.

FWIW, there's also a node module I pulled in as a dev dependency for WIP on the packaging for Windows that makes it easy to do recursive copying. https://github.com/timkendrick/recursive-copy.